src/main/java/seedu/address/model/patient/Patient.java

```diff
@@ -44,8 +44,8 @@ public Patient(Name name, Phone phone, Email email, Gender gender, IcNumber icNu
         this.birthday = birthday;
         this.address = address;
         this.tags.addAll(tags);
-        this.assignedDepartment = assignedDepartment;
-        this.record = record;
+        this.assignedDepartment = new AssignedDepartment(); // default Department given
```

Comment:
Nice! Just to note that we might be changing this such that the Patient is immediately assigned to the Emergency Department upon creation so that it is reflected in the database straight away and the doctor won't have to manually assign a newly admitted patient to the emergency department. The default department is displayed as "-" as a string and is meant to be used for inactive patients, but for now we can just assign it to newly instantiated patient objects.

Reply:
Hi, have taken note of this, thanks!